packages/module/src/DataView/DataView.tsx

```diff
@@ -1,15 +1,26 @@
 import React from 'react';
-import { Text } from '@patternfly/react-core';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { DataViewPaginationProps } from '../Hooks';
 
-export interface DataViewProps {
-  /** Example prop */
-  text?: string;
+export interface DataViewProps extends DataViewPaginationProps {
+  /** Content rendered inside the data view */
+  children?: React.ReactNode;
```

Comment:
This should be mandatory. The view serves no purpose if there are no children.

Also, you should use `React.PropsWithChildren` generic so you don't have to add types for children (when they are optional).